core/src/main/scala/org/apache/spark/sql/delta/commands/CreateDeltaTableCommand.scala

```diff
@@ -317,7 +318,8 @@ case class CreateDeltaTableCommand(
           path, tableDesc.partitionColumnNames, existingMetadata.partitionColumns)
       }
 
-      if (tableDesc.properties.nonEmpty && tableDesc.properties != existingMetadata.configuration) {
+      if (tableDesc.properties.nonEmpty &&
```

Comment:
what happens if I specify some of my table properties with correct capitalization and others lower-case? won't this fail when it should pass?

Reply:
@Pluies please add tests for this change. you can add them to `DeltaTableCreationTests.scala`.

https://github.com/delta-io/delta/blob/master/core/src/test/scala/org/apache/spark/sql/delta/DeltaTableCreationTests.scala